src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/on_system_request_notification_test.cc

```diff
@@ -154,7 +160,7 @@ TEST_F(OnSystemRequestNotificationTest, Run_HTTPType_SUCCESS) {
             (*msg)[strings::params][strings::protocol_version].asInt());
 }
 
-TEST_F(OnSystemRequestNotificationTest, Run_InvalidApp_NoNotification) {
+TEST_F(OnSystemRequestNotificationTest, DISABLED_Run_InvalidApp_NoNotification) {
```

Comment:
why Disabled for all functions in this change?

Reply:
@yang1070 this was regression after rebase on develop, already fixed this tests 
b20a6b1